plugins/rapture/nexus-rapture-plugin/src/main/resources/static/rapture/NX/controller/SettingsForm.js

```diff
@@ -125,6 +128,24 @@ Ext.define('NX.controller.SettingsForm', {
   },
 
   /**
+   * Discard any changes to this form
+   *
+   * @private
+   */
+  discardChanges: function (button) {
+    var form = button.up('form'),
+        record = form.getRecord();
+
+    if (form.api && form.api.load) {
+      form.fireEvent('load', form);
+    }
+    else {
+      form.getForm().reset();
+      form.isValid();
```

Comment:
This is @adreghiciu’s code. I bet we can remove that with no ill effects though.